package.json

```diff
@@ -67,8 +67,9 @@
     "build-app": "tsc && yarn save-build-info && vite build && rm build/config.json",
     "build-packages": "yarn workspaces foreach --include '@terra-ui-packages/*' --parallel --topological-dev run build",
     "lint": "eslint --fix --max-warnings=0 .",
-    "optimize-image-svgs": "svgo --enable=inlineStyles,prefixIds --config '{ \"plugins\": [ { \"inlineStyles\": { \"onlyMatchedOnce\": false } }] }' --pretty -f src/images -r -p 1 --multipass",
-    "preinstall": "node .hooks/check-engine-light.js",
+    "optimize-image-svgs": "yarn optimize-src-image-svgs && yarn optimize-components-image-svgs",
+    "optimize-src-image-svgs": "svgo --enable=inlineStyles,prefixIds --config '{ \"plugins\": [ { \"inlineStyles\": { \"onlyMatchedOnce\": false } }] }' --pretty -f src/images -r -p 1 --multipass",
+    "optimize-components-image-svgs": "svgo --enable=inlineStyles,prefixIds --config '{ \"plugins\": [ { \"inlineStyles\": { \"onlyMatchedOnce\": false } }] }' --pretty -f packages/components/src/icons -r -p 1 --multipass",    "preinstall": "node .hooks/check-engine-light.js",
```

Comment:
Slight preference... I think that would be cleaner. Then `svgo` has to be added to added to components' dev dependencies as well.

Reply:
I do want to keep it so that a single command optimizes both sets of icons. Will I be able to do that if the command for the package icons is not here?

Reply:
Yes

Instead of
```
optimize-image-svgs": "yarn optimize-src-image-svgs && yarn optimize-components-image-svgs"
```

something like
```
optimize-image-svgs": "yarn optimize-src-image-svgs && yarn workspace '@terra-ui-packages/components' run optimize-icon-svgs"
```